### ISSUE TITLE
[binding-request] breqn.sty

### ISSUE THREAD

**Issue by matteosecli** [brucemiller/LaTeXML#944]
Nice package that lets you break long equations.

**Comment by brucemiller** [comment]
Yeah, it is a nice package; I use an older/hacked version in DLMF.

Not the same api (dmath environments and such), but have you looked at the mostly undocumented LineBreaker?  This does basically the same thing as breqn (at least for LaTeXML).  In latexmlpost (or, I assume, latexmlc)
```
--linelength=80
```
to linebreak the displayed math fairly aesthetically.

**Comment by matteosecli** [comment]
> Not the same api (dmath environments and such), but have you looked at the mostly undocumented LineBreaker? This does basically the same thing as breqn (at least for LaTeXML).

Wow, I clearly missed that option while reading the docs! Looks great, I think I'm gonna (conditionally for LaTeXML) redefine `dmath` as just `equation` and then use the `--linelength` option. 😃
